docs/pages/enroll-resources/kubernetes-access/controls.mdx

```diff
@@ -481,40 +481,132 @@ spec:
 ### `kubernetes_resources`
 
 The `kubernetes_resources` field enables a Teleport role to configure access to
-specific resources in a Kubernetes cluster:
+specific resources in a Kubernetes cluster.
+
+The value of this field is a list of mappings, where each mapping is described as follows:
+
+#### Role V6
+
+<Admonition type="tip">
+Role V6 introduced the support for the `kubernetes_resources` field, but was restricted
+to only allow access to pods with kind 'pod'. Later role versions extended the support.
+</Admonition>
 
 ```yaml
 kind: role
 metadata:
   name: kube-access
-version: v8
+version: v6
 spec:
   allow:
     kubernetes_labels:
-      '*':'*'
+      "*": "*"
     kubernetes_resources:
-      - kind: pods
-        namespace: "production"
-        name: "webapp"
-        verbs: ['*']
-        api_group: ""
+      - kind: pod
+        namespace: production
+        name: webapp
     # ...
 ```
 
-The value of this field is a list of mappings, where each mapping has four
-fields:
+- `kind`: The kind of resource to enable access to. Currently, Teleport only
+  supports the value `pod`.
+- `namespace`: The Kubernetes namespace in which to allow access to a resource.
+  In the `kube-access` role, we are allowing access to a pod in the
+  `production` namespace.
+- `name`: The name of the pod to allow access to. In `kube-access`, this is
+  the `webapp` pod.
+
+For both the `namespace` and `name` fields, you can add a wildcard character
+(`*`) to replace any sequence of characters. For example, `name: "pod-*-*"`
+matches pods named `pod-1-a` and `pod-2-c`. As with `kubernetes_labels`, if a
+value begins with `^` and ends in `$`, the Kubernetes Service will treat it as a
+regular expression using Go's `re2` syntax (see the `re2`
+[README](https://github.com/google/re2/wiki/Syntax)).
```

Comment:
Not sure we allow external links in our docs.

Reply:
It is a copy/paste from the other version,